src/main/resources/com/google/api/codegen/java/test.snip

```diff
@@ -272,9 +272,15 @@
 
   @join assert : test.asserts
     @if assert.hasExpectedValueTransformFunction
-      Assert.assertEquals({@assert.expectedValueTransformFunction}(\
-        {@assert.expectedValueIdentifier}), \
-        actualRequest.{@assert.actualValueGetter}());
+      @if assert.hasActualValueTransformFunction
```

Comment:
Or at least have the generator output text that doesn't compile, warning about the unhandled case.